packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -769,19 +769,17 @@ class AppInstance {
 
 /// A [Logger] which sends log messages to a listening daemon client.
```

Comment:
Re: two loggers, that's totally doable - and I do agree that the current state is a bit confusing.

The main value this class adds is letting IDEs convert the spinner progress info available in the CLI to progress mechanisms that make more sense in the IDE (indeterminate progress bars). And what I'm trying to do w/ this change is to plumb more of the verbose info through, so people can use the equivalent of `flutter run -v` to help diagnose when there are issues launching apps.

> The actual change (from logToStdout to having an explicit parent) seems like an improvement

lgtm then, or should I work on specialized _AppRunLogger subclasses?